app/components/dynamictable/dynamictable.html

```diff
@@ -36,6 +37,7 @@
         ng-repeat="project in projects | orderBy:'+projectName' | filter:{projectName:searchQuery} | filter:{category:searchCategory || ''} as filtered">
         <td><a href="{{ project.projectRepository }}">{{ project.projectName }}</a></td>
         <td><a href="{{ project.projectWebsite }}">{{ project.projectWebsite }}</a></td>
+        <td><a href="{{ project.twitterURL }}">{{ project.twitterHandle }}</a></td>
```

Comment:
This looks great!

Reply:
@bproffitt Do you have any opposition to adding a Twitter column?

Reply:
Has this been discussed yet?

Reply:
I'll reach out to him directly; I think at one time, he wasn't getting GitHub notifications.